test/plugins/scenarios/goframe/main.go

```diff
@@ -0,0 +1,51 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package main
+
+//go:nolint
+import (
+	"context"
+	_ "github.com/apache/skywalking-go"
+	"github.com/gogf/gf/v2/frame/g"
+	"github.com/gogf/gf/v2/net/ghttp"
+	"github.com/gogf/gf/v2/os/gcmd"
+	"github.com/gogf/gf/v2/os/gctx"
+	"net/http"
+)
+
+func main() {
+	command := gcmd.Command{
+		Name:  "goframe",
+		Usage: "goframe",
+		Brief: "start http server",
+		Func: func(ctx context.Context, parser *gcmd.Parser) (err error) {
+			s := g.Server()
+			s.SetPort(8080)
+			s.BindHandler("/provider", func(r *ghttp.Request) {
```

Comment:
Can you enrich this case? For example, the plugin test first accesses `/consumer`, then `/consumer` accesses its own `/provider` service. I hope to verify whether the trace context can be passed through during process communication. You can refer to other HTTP use cases.

Reply:
Yes, it's possible. It's essentially a wrapper around net/http, so you should be able to use existing net/http plugins to implement client instrumentation.

Reply:
However, I have added an example of how to use goframe as a client to initiate requests.